spring-cloud-azure-starters/spring-cloud-starter-azure-appconfiguration-config/README.md

```diff
@@ -60,11 +60,11 @@ spring.cloud.azure.appconfiguration.managed-identity.client-id | Client id of th
 
 Name | Description | Required | Default
 ---|---|---|---
-spring.cloud.azure.appconfiguration.stores[0].endpoint | Endpoint of the configuration store, required when `connection-string` is empty. If `connection-string` is empty and application is deployed on Azure VM or App Service with managed identity enabled, will try to load `connection-string` from Azure Resource Management. | Conditional | null
+spring.cloud.azure.appconfiguration.stores[0].endpoint | Endpoint of the configuration store, required when `connection-string` is empty. If `connection-string` is empty alternate connection methods will be attempted based on configuration, such as System Assigned Identity, or User Assigned Identity if the client-id value is set. | Conditional | null
 spring.cloud.azure.appconfiguration.stores[0].prefix | The prefix of the key name in the configuration store, e.g., /my-prefix/application/key.name | No |  null
-spring.cloud.azure.appconfiguration.stores[0].connection-string | Required when `name` is empty, otherwise, can be loaded automatically on Azure Virtual Machine or App Service | Conditional | null
+spring.cloud.azure.appconfiguration.stores[0].connection-string | Required when `endpoint` is empty, otherwise, can be loaded automatically on Azure Virtual Machine or App Service | Conditional | null
```

Comment:
Similarly, reword this and focus on explaining when the connection-string should be used, for example,

*When the connection-string of an App Configuration store is specified, HMAC authentication will be used to connect to the App Configuration service. Errors (what error?) will be thrown if the endpoint and connection-string are specified at the same time.*